etc/profile-a-l/librewolf.profile

```diff
@@ -19,16 +19,40 @@ whitelist ${HOME}/.librewolf
 #noblacklist ${HOME}/.mozilla
 #whitelist ${HOME}/.mozilla
 
+# Uncomment or put in your librewolf.local one of the following whitelist to enable KeePassXC Plugin
+# NOTE: start KeePassXC before Librewolf and keep it open to allow communication between them
+#whitelist ${RUNUSER}/kpxc_server
+#whitelist ${RUNUSER}/org.keepassxc.KeePassXC.BrowserServer
+
 whitelist /usr/share/doc
-whitelist /usr/share/gnome-shell/search-providers/firefox-search-provider.ini
 whitelist /usr/share/gtk-doc/html
 whitelist /usr/share/mozilla
 whitelist /usr/share/webext
 
-# librewolf requires a shell to launch on Arch. We can possibly remove sh though.
-#private-bin bash,dbus-launch,dbus-send,env,librewolf,python*,sh,which
+# librewolf requires a shell to launch on Arch.
+#private-bin bash,dbus-launch,dbus-send,env,librewolf,sh,which
+# Fedora use shell scripts to launch librewolf, at least this is required
+#private-bin basename,bash,cat,dirname,expr,false,librewolf,getenforce,ln,mkdir,pidof,restorecon,rm,rmdir,sed,sh,tclsh,true,uname
```

Comment:
I don't see a Fedora package of librewolf (https://librewolf-community.gitlab.io/install/) ... AppImage and Flatpak seem to be the only install option for (non hacker) fedora users ... remove this.